opencti-platform/opencti-front/src/components/ExportButtons.jsx

```diff
@@ -168,7 +169,10 @@ class ExportButtons extends Component {
       containerId,
       investigationAddFromContainer,
       history,
+      location,
+      workspace,
```

Comment:
this component would need some day a refactoring, the properties we inject are so mixed (csv, investigation, history, workspace ...)

Reply:
If we finish the feature (the 3 slots) completely for the due date we will think about it, but for now, it's not prior